website/docs/r/appautoscaling_policy.html.markdown

```diff
@@ -46,10 +80,32 @@ The following arguments are supported:
 
 * `name` - (Required) The name of the policy.
 * `policy_type` - (Optional) For DynamoDB, only `TargetTrackingScaling` is supported. For any other service, only `StepScaling` is supported. Defaults to `StepScaling`.
-* `resource_id` - (Required) The resource type and unique identifier string for the resource associated with the scaling policy. For Amazon ECS services, this value is the resource type, followed by the cluster name and service name, such as `service/default/sample-webapp`. For Amazon EC2 Spot fleet requests, the resource type is `spot-fleet-request`, and the identifier is the Spot fleet request ID; for example, `spot-fleet-request/sfr-73fbd2ce-aa30-494c-8788-1cee4EXAMPLE`.
-For DynamoDB tables, this value is `table/nameOfTheTable`.
-* `scalable_dimension` - (Required) The scalable dimension of the scalable target. The scalable dimension contains the service namespace,   resource  type, and scaling property, such as `ecs:service:DesiredCount` for the desired task count of an Amazon ECS service, or `ec2:spot-fleet-request:TargetCapacity` for the target capacity of an Amazon EC2 Spot fleet request.
-* `service_namespace` - (Required) The AWS service namespace of the scalable target. Valid values are `ecs` for Amazon ECS services, `ec2` for Amazon EC2 Spot fleet requests and `dynamodb` for DynamoDB tables.
+* `resource_id` - (Required) The resource type and unique identifier string for the resource associated with the scaling policy. A non-comprehensive list of examples for this value:
+
+Resource Type | Resource ID Format | Example
+------------- | ------------------ | -------
+DynamoDB Index | `table/TABLE/index/INDEX` | `table/nameOfTheTable/index/nameOfTheIndex`
+DynamoDB Table | `table/TABLE` | `table/nameOfTheTable`
+EC2 Spot Fleet Request | `spot-fleet-request/ID` | `spot-fleet-request/sfr-73fbd2ce-aa30-494c-8788-1cee4EXAMPLE`
+ECS Service | `service/CLUSTER/SERVICE` | `service/default/sample-webapp`
+EMR Cluster Instance Group | `instancegroup/CLUSTER/GROUP` | `instancegroup/j-2EEZNYKUA1NTV/ig-1791Y4E1L8YI0`
```

Comment:
I admit this confused me for a while when working on DynamoDB autoscaling too, so thanks for mentioning it here. I just wonder - is there any similar table we can link to in the AWS docs instead of keeping our own here?